azkaban-web-server/src/web/js/azkaban/view/schedule-options.js

```diff
@@ -349,15 +349,24 @@ azkaban.AdvancedScheduleView = Backbone.View.extend({
     var period = $('#advperiod').val() + $('#advperiod_units').val();
 
     var flowOverride = {};
+    var jobOverride = {};
```

Comment:
Is this file, `schedule-options.js` ever used? I couldn't find any references to this in the repo:
https://github.com/azkaban/azkaban/search?q=schedule-options

I can make a separate PR to delete this file if you agree that it's not used.

Reply:
I also don't think `schedule-options.js` or `scheduleoptionspanel.vm` are used. Feel free to submit their removal.

Reply:
Here it is: https://github.com/azkaban/azkaban/pull/2755